packages/devtools_app/lib/src/screens/inspector/diagnostics.dart

```diff
@@ -58,10 +59,56 @@ class DiagnosticsNodeDescription extends StatelessWidget {
     );
   }
 
-  Iterable<TextSpan> _buildDescriptionTextSpans(
+  /// Approximates the width of the elements inside a [RemoteDiagnosticsNode]
+  /// widget.
+  static double approximateNodeWidth(
+    RemoteDiagnosticsNode? diagnostic,
+  ) {
+    final spans = DiagnosticsNodeDescription.buildDescriptionTextSpans(
+      diagnostic?.description ?? '',
+      const TextStyle(),
```

Comment:
general concern for using a blank TextStyle is font sizing. Have you tried this with different sizes of text?

Reply:
Good catch on this, it turns out it effects the sizing!